docs/eventing/getting-started.md

```diff
@@ -19,299 +21,377 @@ Use this guide to learn how to create, send, and verify events in Knative. The s
-          source: sendoff
-      subscriber:
-        ref:
-         apiVersion: v1
-         kind: Service
-         name: goodbye-display
-    END
-    ```
-
-   The command creates a trigger that sends all events of source `sendoff` to your event consumer named `goodbye-display`.
-
-1. Verify that the triggers are working correctly by running the following command:
-
-    ```sh
-    kubectl --namespace event-example get triggers
-    ```
-
-    This returns the `hello-display` and `goodbye-display` triggers that you created:
-
-    ```sh
-    NAME                   READY   REASON   BROKER    SUBSCRIBER_URI                                                                 AGE
-    goodbye-display        True             default   http://goodbye-display.event-example.svc.cluster.local/                        9s
-    hello-display          True             default   http://hello-display.event-example.svc.cluster.local/                          16s
-    ```
-
-    If the triggers are correctly configured, they will be ready and pointing to the correct **Broker** (the default broker)  and **SUBSCRIBER_URI** (triggerName.namespaceName.svc.cluster.local). If this is not the case, see the [Debugging Guide](./debugging/README.md) to help troubleshoot the issue.
-
-You have now created all of the resources needed to receive and manage events. You created the `Broker`, which manages the events sent to event consumers with the help of triggers.  In the next section, you will make the event producer that will be used to create your events.
+   ```sh
+   kubectl --namespace event-example apply --filename - << END
+   apiVersion: eventing.knative.dev/v1alpha1
+   kind: Trigger
+   metadata:
+     name: goodbye-display
+   spec:
+     broker: default
+     filter:
+       attributes:
+         source: sendoff
+     subscriber:
+       ref:
+        apiVersion: v1
+        kind: Service
+        name: goodbye-display
+   END
+   ```
+
+   The command creates a trigger that sends all events of source `sendoff` to
+   your event consumer named `goodbye-display`.
+
+1. Verify that the triggers are working correctly by running the following
+   command:
+
+   ```sh
+   kubectl --namespace event-example get triggers
+   ```
+
+   This returns the `hello-display` and `goodbye-display` triggers that you
+   created:
+
+   ```sh
+   NAME                   READY   REASON   BROKER    SUBSCRIBER_URI                                                                 AGE
+   goodbye-display        True             default   http://goodbye-display.event-example.svc.cluster.local/                        9s
+   hello-display          True             default   http://hello-display.event-example.svc.cluster.local/                          16s
+   ```
+
+   If the triggers are correctly configured, they will be ready and pointing to
+   the correct **Broker** (the default broker) and **SUBSCRIBER_URI**
+   (triggerName.namespaceName.svc.cluster.local). If this is not the case, see
```

Comment:
might want to add a note that svc.cluster.local might differ depending on the actual cluster installation?